src/core/hle/kernel/thread.h

```diff
@@ -43,12 +43,13 @@ class IPCCallback;
 class Thread : public WaitObject {
 
 public:
-    enum Status : u32 {
+    enum class Status : u32 {
         Running,
         Ready,
         WaitSleep,
         WaitSyncAll,
         WaitSyncAny,
+        WaitHleEvent,
```

Comment:
```suggestion
        WaitHLEEvent,
```

given `IPC` is already used below it.